lib/pbench/test/unit/server/test_schema.py

```diff
@@ -68,21 +82,27 @@ def ok(auth: Auth, username: str) -> User:
             (ParamType.DATE, 1),  # not a string representing a date
             (ParamType.USER, "drb"),  # we haven't established a "drb" user
             (ParamType.USER, False),  # not a user string
-            (ParamType.ACCESS, "foobar"),  # ACCESS is "public" or "private"
+            (ParamType.ACCESS, ["foobar"]),  # ACCESS is "public" or "private"
             (ParamType.ACCESS, 0),  # ACCESS must be a string
-            (ParamType.LIST, "foobar"),  # Not a list
-            (ParamType.LIST, 0),  # Not a list
         ),
     )
     def test_failed_conversions(self, test, monkeypatch):
+        """
+        Test unsuccessful parameter conversion / normalization.
+
+        NOTE that we can't test KEYWORD here without a Parameter to define the
+        set of keywords; we'll test that separately.
+        """
+
         def not_ok(auth: Auth, username: str) -> User:
             raise UnknownUser()
 
         monkeypatch.setattr(Auth, "verify_user", not_ok)
 
         ptype, value = test
+        param = Parameter("test", ptype)
         with pytest.raises(ConversionError) as exc:
-            ptype.convert(value)
+            ptype.convert(value, param)
         assert str(exc).find(str(value))
```

Comment:
Since you're interested in a boolean result, perhaps
```
        assert str(value) in str(exc)
```

Reply:
Ditto line 208 and possibly elsewhere.

Reply:
Yes, I keep forgetting Python lets you do that ...

Reply:
Yeah...it's annoyingly sleek.